pdns/dnsdist-lua.cc

```diff
@@ -1713,7 +1713,9 @@ static void setupLuaConfig(LuaContext& luaCtx, bool client, bool configCheck)
     return pool;
   });
 
+  luaCtx.writeFunction("setVerbose", [](bool verbose) { g_verbose = verbose; });
```

Comment:
should this also run in `dnsdist -c`?

Reply:
I don't see any drawback right away from being able to switch the verbose mode from the configuration when in client mode. Of course it will not apply to the current process but to the one we connect to if the command is ussed at runtime.

Reply:
:+1: